src/io/sudhir/switchboard/boards/Board.java

```diff
@@ -19,6 +20,8 @@ static Board create(Collection<? extends Supply> supplies, Collection<? extends
 
   boolean canProceed();
 
+  ImmutableSet<Supply> availableSupplies();
```

Comment:
Switch to Set, don't expose Immutables.

Reply:
Shall I switch it to a `Stream<Supply>`?

Reply:
No, that's too much drama. Set is fine.

Reply:
Use a Stream for objects are likely to be lazily computed.